core/util/makeConfig.js

```diff
@@ -28,36 +40,39 @@ function makeConfig (customConfig) {
   // Continuous Integration (CI) report
   config.ci_report = config.customBackstop + '/ci_report';
 
+  // HTML Report
+  config.html_report = config.customBackstop + '/html_report';
```

Comment:
Yes I think we can change that

Get Outlook for iOS




On Sat, Aug 20, 2016 at 1:57 AM +0200, "Garris" <notifications@github.com> wrote:












In core/util/makeConfig.js:
> @@ -28,36 +40,39 @@ function makeConfig (customConfig) {
>    // Continuous Integration (CI) report
>    config.ci_report = config.customBackstop + '/ci_report';
>  
> +  // HTML Report
> +  config.html_report = config.customBackstop + '/html_report';



Can we default to backstop_data/html_report?



—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.